qlib/linux_def.rs

```diff
@@ -3266,6 +3267,31 @@ pub struct LibcStat {
     pub pad: [i64; 3],
 }
 
+#[cfg(target_arch = "aarch64")]
+#[derive(Debug, Default, Copy, Clone)]
+#[repr(C)]
+pub struct LibcStat {
```

Comment:
isn't this the same as done in https://github.com/QuarkContainer/Quark/pull/1015/commits/e1469ade75fb7ae331e7d2ef562a2f7ef0a4dd79 ?